__test__/FilterSidebarMultiselectDropdownFacetGroup.test.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
```

Comment:
All the comments from `FilterSidebarCheckboxFacetGroup.test.js` apply here as well.